docker/Dockerfile

```diff
@@ -23,7 +45,7 @@ RUN cd /opt/lmdeploy &&\
     cmake .. \
         -DCMAKE_BUILD_TYPE=RelWithDebInfo \
         -DCMAKE_EXPORT_COMPILE_COMMANDS=1 \
-        -DCMAKE_INSTALL_PREFIX=/opt/tritonserver \
+        -DCMAKE_INSTALL_PREFIX=/opt/lmdeploy/install \
```

Comment:
Can we substitute "cmake .." snippet by the following command? @lzhangzz 

```shell
../generate.sh make
make -j$(nproc) && make install
```
It helps guide users to build lmdeploy from source in the installation guide, referring to #2038 

`generate.sh` should be changed

Reply:
ok

Reply:
may make the `generate.sh` file executable